Documentation/faq.md

```diff
@@ -78,6 +78,10 @@ On the other hand, if the downed member is removed from cluster membership first
 
 etcd sets `strict-reconfig-check` in order to reject reconfiguration requests that would cause quorum loss. Abandoning quorum is really risky (especially when the cluster is already unhealthy). Although it may be tempting to disable quorum checking if there's quorum loss to add a new member, this could lead to full fledged cluster inconsistency. For many applications, this will make the problem even worse ("disk geometry corruption" being a candidate for most terrifying).
 
+### Why does etcd lose its leader from disk latency spikes?
+
+Disk latency is part of leader liveness. Suppose the cluster leader takes a minute to fsync a raft log update to disk, but the etcd cluster has a one second election timeout. Even though the leader can send network requests within the election interval (e.g., respond to heartbeats), it's effectively unavailable because it can't commit any new proposals; it's waiting on the slow disk. If the cluster frequently loses its leader due to disk latencies, try [tuning][tuning] the disk settings or etcd time parameters.
```

Comment:
There's nothing stopping it from send heartbeats, but it shouldn't send any heartbeats in this situation (i.e., `the leader can send network requests` and clearly etcd doesn't send anything since this is an FAQ explanation for this sort of leader election). Suppose fsync blocks forever on the leader-- it'll block the cluster from making any progress if the leader keeps sending heartbeats.